subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -1393,6 +1395,11 @@ uint8_t ull_adv_aux_chm_update(void)
 		aux->chm[chm_last].data_chan_count =
 			ull_chan_map_get(aux->chm[chm_last].data_chan_map);
 		aux->chm_last = chm_last;
+
+		if (!aux->is_started) {
+			/* Ticker not started yet, apply new channel map now */
+			aux->chm_first = aux->chm_last;
+		}
```

Comment:
Do not modify `chm_first` in ULL execution context, instead modify only `chm_last`

Reply:
As far as I remember, this was done to handle the case where we get two channel updates without the advertising running. Currently, that will fail due to this check earlier in the function:
```
		if (aux->chm_last != aux->chm_first) {
			/* TODO: Handle previous Channel Map Update being in
			 * progress
			 */
			continue;
		}
```
Shouldn't it be safe to update `chm_first` when the ticker is not running?

Reply:
Could you add a comment in the code explaining the same that it is safe to update the indices as the advertising is not currently running.

Reply:
I have added a comment about that